### ISSUE TITLE
VoiceOver Accessibility Issue

### ISSUE THREAD

**Issue by AnonymousAccessibilityAdvocate** [session-foundation/session-ios#715]
The menu or account menu should have a button element labeled for the screen reader to read it.
